packages/elvis-dropdown-lit/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@elvia/elvis-dropdown-lit",
+  "version": "1.0.0",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "type": "module",
+  "files": [
+    "dist",
+    "react.ts"
+  ],
+  "scripts": {
+    "build": "tsup src/index.ts --dts --format esm --out-dir dist",
+    "start": "tsup src/index.ts --sourcemap --dts --format esm --out-dir dist --watch"
```

Comment:
So clean and easy to read ❤️ And as I hoped (and hope we can keep moving forwards); no custom build. Could we do this with `esbuild` and skip the typings? Or are they intended for React users?

Reply:
We must also think about what to do with the CEG again (considering we don't export any prop interfaces). Maybe there exists a way of extracting the props from the custom element and infer a type from it that we can feed into the existing types for the CEG?

Reply:
I agree, super nice and clean!
I believe the typings are used for React, and also for anyone who does a `document.querySelector('')` or even when just looking at the type of the custom element tag in Angular. 
![image](https://github.com/3lvia/designsystem/assets/38653456/518ae757-8c62-45c3-b9cb-6eb1ac993eee)
Here I am just hovering the element, and it sees what type it has! This is due to the 
`declare global {
  interface HTMLElementTagNameMap {
    'e-dropdown': EDropdown;
  }
}`

Reply:
We should be able to access the prop interfaces from the React-component through something like `type Props = PropsWithoutRef<typeof EDropdownComponent>` 🤔